src/theme/MuiThemeProvider/muiTheme.ts

```diff
@@ -84,42 +84,50 @@ export default createTheme({
     color: defaultTheme.palette.text.primary,
     h1: {
       fontSize: '28px',
-      lineHeight: '42px',
+      // line-height has default value 1.5 for all font-sizes. can be defined in body
+      // lineHeight: '1.5',
     },
     h2: {
       fontSize: '25px',
-      lineHeight: '37px',
+      // line-height has default value 1.5 for all font-sizes. can be defined in body
+      // lineHeight: '1.5',
     },
     h3: {
       fontSize: '20px',
-      lineHeight: '30px',
+      // line-height has default value 1.5 for all font-sizes. can be defined in body
+      // lineHeight: '1.5',
     },
     h4: {},
     h5: {},
     h6: {},
     subtitle1: {
       fontSize: '17px',
-      lineHeight: '25px',
+      // line-height has default value 1.5 for all font-sizes. can be defined in body
+      // lineHeight: '1.5',
     },
     subtitle2: {},
     body1: {
       fontSize: '16px',
-      lineHeight: '24px',
+      // line-height has default value 1.5 for all font-sizes. can be defined in body
+      // lineHeight: '1.5',
     },
     body2: {},
     small1: {
       fontSize: '0.875rem',
-      lineHeight: '1.5',
+      // line-height has default value 1.5 for all font-sizes. can be defined in body
+      // lineHeight: '1.5',
       fontWeight: 600,
     },
     small2: {
       fontSize: '0.875rem',
-      lineHeight: '1.5',
+      // line-height has default value 1.5 for all font-sizes. can be defined in body
+      // lineHeight: '1.5',
       fontWeight: 400,
     },
     button: {
       fontSize: '14px',
-      lineHeight: '21px',
+      // line-height has default value 1.5 for all font-sizes. can be defined in body
+      // lineHeight: '1.5',
```

Comment:
Let's move these typography changes to a separate PR, so that we don't try to do everything at once.

Reply:
done